snapm/manager/_manager.py

```
@@ -351,6 +364,196 @@
+    path = ""
+    i = 0
+    while i < len(escaped):
+        if escaped[i] == "\\":
+            if escaped[i + 1] == "\\":
+                path += "\\"
+                i += 1
+            elif escaped[i + 1] == "x":
+                path += decode_byte(escaped[i + 2 : i + 4])
+                i += 3
+            else:
+                raise ValueError(f"Invalid escape sequence: {escaped[i: i + 2]}")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)